info.toml

```diff
@@ -614,6 +614,24 @@ hint = """
 
 # STANDARD LIBRARY TYPES
 
+[[exercises]]
+name = "box1"
+path = "exercises/standard_library_types/box1.rs"
+mode = "compile"
+hint = """
+Step 1
+The compiler's message should help: since we cannot store the value of the actual type
+when working with recursive types, we need to store a reference (pointer) to its value.
+We should therefore place our `List` inside a `Box`. More details in the book here:
+https://doc.rust-lang.org/book/ch15-01-box.html#enabling-recursive-types-with-boxes
+
+Step 2
+Creating an empty list should be fairly straightforward (hint: peek at the assertions).
+For a non-empty list keep in mind that wee want to use our Cons "list builder".
```

Comment:
wee -> we

Reply:
good spot, macbook butterfly keyboard 😅 . I'll correct this and put the hints through Grammarly